### DISCUSSION TITLE
Welcome to TimeSchedule-Mobile-Group11 Discussions!

### DISCUSSION THREAD

**Discussion by tsdevtool** [tsdevtool/TimeSchedule-Mobile-Group11#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tsdevtool** [comment]
Hello

**Comment by Kcode1A** [comment]
sắp làm xong phần user case diagram.

**Reply by Kcode1A** [reply]
đã hoàn thành xong cơ bản User case và Activity Digram.
